shared/src/EMBC.Utilities/Transformation/Transformator.cs

```diff
@@ -25,6 +25,21 @@ public class HbsTransformator : ITransformator
     {
         public async Task<TransformationResult> Transform(TransformationData transformationData)
         {
+            Handlebars.RegisterHelper("customComparerDecimalAmount", (writer, context, parameters) =>
+            {
+                if (parameters.Length < 3)
```

Comment:
what is this magic number for? what happens if there's email template with less? The transformator is used by other functionality as well, this is very well baked to what you're developing and makes no sense here